pkgs/desktops/plasma-5/plasma-systemmonitor.nix

```diff
@@ -0,0 +1,33 @@
+{ mkDerivation
+, lib
+, extra-cmake-modules
+, kdoctools
+, qtquickcontrols2
+, kconfig
+, kcoreaddons
+, ki18n
+, kiconthemes
+, kitemmodels
+, kitemviews
+, knewstuff
+, ksysguard
+, libksysguard
+, qtbase
+}:
+
+mkDerivation {
+  name = "plasma-systemmonitor";
+  meta.broken = lib.versionOlder qtbase.version "5.15.0";
```

Comment:
This is missing description, homepage, maintainer and license.

Reply:
No, this is how the plasma stuff is handled.

Reply:
If you take a look at the search website https://search.nixos.org/packages?channel=20.09&show=plasma5.kde-gtk-config&from=0&size=50&sort=relevance&query=plasma you can see that a lot of plasma packages miss all those infos which is not great.